pyfolio/utils.py

```diff
@@ -259,3 +260,45 @@ def extract_rets_pos_txn_from_zipline(backtest):
     transactions.index.tz = None
 
     return returns, positions, transactions, gross_lev
+
+
+SETTINGS = {
+    # Settings dict to store functions/values that may
+    # need to be overridden depending on the users environment
+    'returns_func': default_returns_func
+}
+
+
+def register_return_func(func):
+    """
+    Registers the function that will be called for
+    retrieving returns data.
+
+    Parameters
+    ----------
+    func : function
+        A function that accepts a symbol and
+        returns a pandas.Series of asset returns.
+
+    Returns
+    -------
+    None
+    """
+    SETTINGS['returns_func'] = func
+
+
+def get_symbol_rets(symbol):
```

Comment:
I realized the same thing a bit ago (see https://github.com/quantopian/pyfolio/pull/92#issuecomment-125677310) I think `functools.partial` will work pretty well to avoid the extra parameters all over. How about one of the methods below? I'm think the second has less potential issues in the future. 

```
def register_return_func(func, default_kwargs):
    SETTINGS['returns_func'] = functools.partial(func, **default_kwargs)

def register_return_func(func, default_kwargs):
    SETTINGS['returns_func'] = functools.partial(func, default_kwargs=default_kwargs)
```